src/message.cpp

```diff
@@ -6,19 +6,23 @@
 
 #include "../include/data_structures/message.hpp"
 
```

Comment:
Also add a comment on starting of this file (on the top). Use 'brief' word to give a summary of what this file does

Reply:
Done. Please check again. Thanks